TempleEight/docs/installation.md

```diff
@@ -0,0 +1,55 @@
+---
+id: installation
+title: Installation Guide
+sidebar_label: Installation Guide
+---
+
+Welcome to the Temple installation guide!
+
+In order to get using Temple, you'll need to install the *Temple CLI*. 
+
+## Prerequisites 
+
+The Temple CLI requires [Java](https://java.com/en/download/help/download_options.xml) to run.
+
+In order to build and orchestrate generated services, you'll need:
+
+* [Docker](https://www.docker.com/)
+* For projects orchestrated with Docker-Compose: 
+    * [docker-compose](https://docs.docker.com/compose/)
+* For projects orchestrated with Kubernetes:
+    * [Minikube](https://minikube.sigs.k8s.io/docs/) for local development
+    * [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#verifying-kubectl-configuration) for managing Kubernetes clusters
+
+
+## Installation Instructions
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
```

Comment:
Importing works mid-markdown? What is this madness?

Reply:
mdx bb